db/controllers/reviews.controller.js

```diff
@@ -17,4 +17,16 @@ exports.selectAllReviews = (request, response, next) => {
     .then((reviews) => {
         return response.set(200).send({reviews})
     })
+}
+
+exports.retrieveComments = (request, response, next) => {
+    const reviewId = request.params.review_id
+    
+    fetchComments(reviewId)
```

Comment:
Please consider using a Promise.all() here to check if the review exists and get the comments at the same time.